packages/eslint-plugin-js/rules/keyword-spacing/keyword-spacing.ts

```diff
@@ -396,15 +418,12 @@ export default {
      * @param {ASTNode} node A node to report.
      * @returns {void}
      */
-    function checkSpacingForForInStatement(node) {
+    function checkSpacingForForInStatement(node: TSESTree.ForInStatement) {
       checkSpacingAroundFirstToken(node)
 
-      const inToken = sourceCode.getTokenBefore(node.right, isNotOpeningParenToken)
-      const previousToken = sourceCode.getTokenBefore(inToken)
-
-      if (previousToken.type !== 'PrivateIdentifier')
```

Comment:
`PrivateIdentifier` is a node, not token, so tsc complains that this comparison is excessive